xml/System.Dynamic/DynamicObject.xml

```diff
@@ -935,7 +1166,7 @@ Dim testExplicit2 = CTypeDynamic(sampleObject, GetType(String))
   
  You can get information about the type of the unary operation by using the `Operation` property of the `binder` parameter.  
   
- If your dynamic object is used only in C# and Visual Basic, the `binder.Operation` property can have one of the following values from the <xref:System.Linq.Expressions.ExpressionType> enumeration. However, in other languages such as IronPython or IronRuby, you can have other values. For more information about <xref:System.Linq.Expressions.ExpressionType>, see documentation on the [CodePlex](http://go.microsoft.com/fwlink/?LinkId=141028) Web site.  
+ If your dynamic object is used only in C# and Visual Basic, the `binder.Operation` property can have one of the following values from the <xref:System.Linq.Expressions.ExpressionType> enumeration. However, in other languages such as IronPython or IronRuby, you can have other values. For more information about <xref:System.Linq.Expressions.ExpressionType>, see [documentation](~/dotnet/api/system.linq.expressions.expressiontype).  
```

Comment:
There are two links to the same topic. Change this to, "For more information, see \<xref::System.Linq.Expressions.ExpressionType>."